runtime/participant-runtime.ss

```diff
@@ -767,3 +767,380 @@
       (hash-ref j
         (cond ((symbol? k) (symbol->string k))
               (else        (string->symbol k)))))))
+
+
+;; ---------------------------------------------------
+;; ------------------ Off-chain communication channels
+;; ---------------------------------------------------
+
+
+;; ------------------ Channel types
+
+
```

Comment:
Is there a better way to structure various channels?
Currently the way I discriminate between channels is by using the `tag` field,
and matching on it.
See `participant-runtime#init-offchain-channel` for an example.

Reply:
Updated.